### ISSUE TITLE
EEXIST: file already exists, mkdir 'tmp/fontawesome-cache_path-1avxyzXv.tmp/node_modules/'

### ISSUE THREAD

**Issue by lougreenwood** [FortAwesome/ember-fontawesome#24]
```
ember-cli: 3.1.0-beta.1
node: 9.8.0
os: darwin x64
```

Just tried migrating from the old `ember-font-awesome` to this new add-on, but I get an error with `ember s`.

Here's the log:
```
  - nodeName: Rollup
  - originalErrorMessage: EEXIST: file already exists, mkdir '/Users/Lou/Sites/prog/tmp/fontawesome-cache_path-1avxyzXv.tmp/node_modules/'
  - stack: Error: EEXIST: file already exists, mkdir '/Users/Lou/Sites/prog/tmp/fontawesome-cache_path-1avxyzXv.tmp/node_modules/'
    at Object.fs.mkdirSync (fs.js:909:18)
    at patches.forEach (/Users/Lou/Sites/prog/node_modules/broccoli-rollup/dist/index.js:68:26)
    at Array.forEach (<anonymous>)
    at Rollup.build (/Users/Lou/Sites/prog/node_modules/broccoli-rollup/dist/index.js:63:17)
    at /Users/Lou/Sites/prog/node_modules/broccoli-plugin/read_compat.js:93:34
    at tryCatch (/Users/Lou/Sites/prog/node_modules/rsvp/dist/rsvp.js:525:12)
    at invokeCallback (/Users/Lou/Sites/prog/node_modules/rsvp/dist/rsvp.js:538:13)
    at publish (/Users/Lou/Sites/prog/node_modules/rsvp/dist/rsvp.js:508:7)
    at flush (/Users/Lou/Sites/prog/node_modules/rsvp/dist/rsvp.js:2415:5)
    at process._tickCallback (internal/process/next_tick.js:112:11)

=================================================================================
```

package.json contains:
```
"@fortawesome/fontawesome-free-solid": "5.1.0-3",
"@fortawesome/ember-fontawesome": "^0.0.12",
```

**Comment by bgentry** [comment]
I just ran into this issue as well. It wasn't caused by upgrading to `0.0.12`, but by blowing away my `yarn.lock` and letting the dependencies re-resolve. Only an issue on macOS, on Linux (in CI) it appears to work fine.

This is an issue even on `0.0.11-4`.

I'm on ember-cli v3.0.1.

**Comment by jrjohnson** [comment]
Looks like this can be repeated easily just run `yarn upgrade` in this addon and `ember test` will spit out the same error.

**Comment by robmadole** [comment]
Ok, so how do I reproduce?

1. Create a new project with `ember new`
1. Install `ember-font-awesome` addon
1. Install `@fortawesome/ember-fontawesome`

**Comment by bgentry** [comment]
Pretty simple, really. Exactly what @jrjohnson said:

1. Clone this repo on master
2. `yarn` to install packages
3. `ember serve`, see that it works
4. `yarn upgrade`
5. `ember serve`, see the errors reported here.

**Comment by robmadole** [comment]
Ok, I've ran this down. I think the root cause is this: https://github.com/chadhietala/broccoli-rollup/issues/39

**Comment by robmadole** [comment]
Ok, #31 should fix this for now. Anyone who is experiencing this may need to `rm -rf node_modules/@fortawesome` and then `yarn` again.

**Comment by krisselden** [comment]
I have fixed the underlying issue in broccoli-rollup, I will publish shortly but I think the fix needs to be pulled into something else

**Comment by robmadole** [comment]
I've released 0.1.0-3 that includes an updated dep of `broccoli-rollup`. According to my testing that fixes it up 👍

**Comment by st-h** [comment]
@robmadole just been running into this issue and can confirm that `0.1.0-3` fixes this. However it would be a lot easier to catch these things if issues wouldn't be closed that early (before the actual fix has been released).
